var/spack/repos/builtin/packages/py-pulsar-galaxy-lib/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyPulsarGalaxyLib(PythonPackage):
+    """Distributed job execution application built for Galaxy (http://galaxyproject.org/)."""
+
+    homepage = "https://github.com/galaxyproject/pulsar"
+    pypi = "pulsar-galaxy-lib/pulsar-galaxy-lib-0.14.16.tar.gz"
+
+    version("0.14.16", sha256="f3330350d2e85c7228cebf83f74fc4c0cc5e8e7557bb6e5ae55f5556d7e6fbff")
+
+    depends_on("py-setuptools", type="build")
+
+    depends_on("py-webob", type=("build", "run"))
+    depends_on("py-psutil", type=("build", "run"))
+    depends_on("py-pastedeploy", type=("build", "run"))
+    depends_on("py-pyyaml", type=("build", "run"))
```

Comment:
Missing deps:
```
galaxy-job-metrics>=19.9.0.dev1
galaxy-objectstore>=19.9.0.dev1
galaxy-tool-util>=19.9.0.dev3
galaxy-util>=19.9.0.dev1
```
Just drop the dev*, Spack sorts dev as newer than not-dev

Reply:
done